test/Msbuild.Tests.Utilities/Msbuild.Tests.Utilities.csproj

```diff
@@ -6,6 +6,9 @@
     <RuntimeFrameworkVersion>$(CLI_SharedFrameworkVersion)</RuntimeFrameworkVersion>
     <AssemblyName>Msbuild.Tests.Utilities</AssemblyName>
     <AssetTargetFallback>$(AssetTargetFallback);dotnet5.4;portable-net451+win8</AssetTargetFallback>
+    <AssemblyOriginatorKeyFile>../../tools/Key.snk</AssemblyOriginatorKeyFile>
```

Comment:
Why was this necessary?

Reply:
Got error about referencing unsigned from signed. I'm at a loss for how it ever worked.